### DISCUSSION TITLE
Introductions (for Hacktoberfest Meetup event)

### DISCUSSION THREAD

**Discussion by kozo2** [Bioconductor/biocblog#22] [General]
Please use this thread to introduce yourselves!

Everyone is welcome!

**Comment by kozo2** [comment]
Hello,

My name is Kozo Nishida.
I am an research software engineer in Tokyo University of Agriculture and Technology (TUAT).

I developed my first Bioconductor package in 2020 (transomics2cytoscape), and continued developing more since (AHPathbankDbs, AHWikipathwaysDbs).

After attending some Bioconductor related meetups in Japan, I've been helping as a co-organiser of Bioconductor online events.
I particularly enjoy the exchange of ideas with other community members.

I've also enjoy working in Python, C#, and other programming languages.
I'm one of the organizers of the PyData meetup in Japan.
http://osaka.pydata.org/
Actually I moved from Osaka to Tokyo this year, but I'm still an organizer of PyData Osaka.
Because the pandemic control policy is still strict in Japan and almost all meetups are still online.
Also, I have been involved in the development of [MS-DIAL](https://pubmed.ncbi.nlm.nih.gov/25938372/) since this year, and I usually write code in C#.
(MS-DIAL is written in C#.)

I am also a Carpentries instructor.
https://carpentries.org/instructors/
I also translate Carpentries teaching materials into Japanese.
https://github.com/swcarpentry-ja
And as a member of [the Bioconductor multilingual working group](http://workinggroups.bioconductor.org/currently-active-working-groups-committees.html#multilingual),
I also hold [the monthly Bioconductor translation-a-thon](https://www.meetup.com/boston-r-bioconductor-for-genomics/events/287802282/).
(We use a translation platform called Crowdin for the translation. https://bioconductor.crowdin.com/ .)

I'm looking forward to joining in the original course effort instead of the translation.

**Comment by JauntyJJS** [comment]
Hi,

I am [Jeremy](https://jeremy-selva.netlify.app/).
I am currently working in the [Singapore Lipidomics Incubator](https://sling.sg/) as a research assistant, creating simple software for the lab such as [MSOrganiser](https://github.com/SLINGhub/MSOrganiser) and R package [DCVtestkit](https://github.com/SLINGhub/DCVtestkit).
